src/Xamarin.Android.Build.Tasks/Tasks/ResolveSdksTask.cs

```diff
@@ -309,6 +309,16 @@ public bool RunTask ()
 				return false;
 			}
 
+			int apiLevel;
+			var augustDeadline = new DateTime (2018, 8, 1);
+			var novemberDeadline = new DateTime (2018, 11, 1);
```

Comment:
Hmm, I don't know about having the deadline. It seems weird to me, for a warning to suddenly appear one day.

Can we just always emit the warning? So people can be aware ahead of time?

Reply:
no sure.. the spec called for specific dates. @jonpryor @JonDouglas ?

Reply:
I'd agree with @jonathanpeppers as to just emitting the warning until they increase the `targetSdkVersion`.

Reply:
@dellis1972: as per @JonDouglas's response above, we should just always emit the warning.  Please remove the `*Deadline` values.